src/fstab-generator/fstab-generator.c

```diff
@@ -781,6 +788,8 @@ static MountPointFlags fstab_options_to_flags(const char *options, bool is_swap)
                 flags |= MOUNT_GROWFS;
         if (fstab_test_option(options, "x-systemd.pcrfs\0"))
                 flags |= MOUNT_PCRFS;
+        if (fstab_test_option(options, "quota\0" "usrquota\0"  "grpquota\0"  "prjquota\0"))
+                flags |= MOUNT_QUOTA;
```

Comment:
hmm, I am pretty sure you ave to check the fstype too here, via fstype_needs_quota(), since only some fs that accept the "quota" mount option actually needs these services.

that's why the old code checked for this.